dev/plugins/hu.elte.txtuml.validation/src/hu/elte/txtuml/validation/messages.properties

```diff
@@ -1,13 +1,12 @@
 InvalidChildrenElement_message=Invalid element in {0} : {1}.
 InvalidDataTypeField_message=Invalid type in data type field. Only boolean, double, int, String, model enums and data types are allowed.
-InvalidDataTypeMethod_message=Data types can only contain constructor methods.
```

Comment:
Should be removed from _[Messages.java](https://github.com/ELTE-Soft/txtUML/blob/c4d09e986a5a93b6eaa86e270eb6b5482de72329/dev/plugins/hu.elte.txtuml.validation/src/hu/elte/txtuml/validation/Messages.java#L9)_ as well.